shelter-in-plates/src/views/Home.vue

```diff
@@ -134,7 +134,4 @@ export default {
 }
 </script>
 <style lang="scss">
-body {
-    margin-left: 0px;
```

Comment:
@shchoi22 this was explicitly set, but I couldn't figure out why.

It was causing the default theme's left gutter (margin-left 30px) to be overwritten.

Tagging you just in case there was an important reason and I should revert this change.